### ISSUE TITLE
Add ability to reflow without rearranging columns

### ISSUE THREAD

**Issue by christiangoudreau** [ArcBees/gwtquery-elastic-plugin#12]
The use case is as follow: A disclosure panel is added to elastic. When the disclosure panel is expended, we don't want the columns to be rearranged.

**Comment by ewolk** [comment]
does this accurately describe the proposed change?
```
elastic-flowed list of 7 ordered elements, expanding element [2]

without no-rearrange:
[1] [2] [3]    =>   [1] [2] [3]
[4] [5] [6]         [4] |_| [5]
[7]                 [6] [7]

with no-rearrange:
[1] [2] [3]    =>   [1] [2] [3]
[4] [5] [6]         [4] [_] [6]
[7]                 [7] [5]
```

**Comment by christiangoudreau** [comment]
Thanks, yes! :D

**Comment by jDramaix** [comment]
Should be fixed by #15

**Comment by jDramaix** [comment]
Can we close this ?
